app/js/helpers/authHelper.js

```diff
@@ -0,0 +1,12 @@
+import axiosInstance from '../config/axiosInstance';
+
+const getLoginStatus = url => new Promise(
+  (resolve, reject) => {
+    axiosInstance
```

Comment:
Nice job. However, is there any need to wrap axios instance in a promise since axios already returns a promise?

Reply:
Hey @emmabaye, you're welcome. Yes there is need. It is always advisable to separate logic to make it easier to test. I created a function that returns a `promise.resolve` or `promise.reject`. Excluding the `new Promise` renders that logic invalid. So after the axiosInstance returns a response, I resolve and return that. Same goes for the error. Also refer to this document to get some more context: https://medium.com/@devlob/recreating-axios-with-es6-promises-815f6b0d1c17.

Reply:
Thanks, @emmabaye, the axiosInstance also works with a slight tweak to how the response is handled. Addressed.